docs/sphinx/_toc.yml.in

```
@@ -1,17 +1,19 @@
 defaults:
   numbered: False
-root: src/index.rst
+root: src/index
 subtrees:
   - entries:
     - file: src/getting-started
     - file: src/concepts
       entries:
-      - file: src/concepts/benchmarking
-      - file: src/concepts/tuning
-      - file: src/concepts/kernels
-      - file: src/concepts/library-creation
+        - file: src/concepts/benchmarking
+        - file: src/concepts/tuning
+        - file: src/concepts/kernels
+        - file: src/concepts/library-creation
     - file: src/api-reference
     - file: src/cli-reference
+      entries: 
+      - file: src/cli-reference/TensileCreateLibrary                    
```

```suggestion
        - file: src/cli-reference/TensileCreateLibrary
```
Match file indentation under entries above.